libs/api/domains/syslumenn/src/lib/client/syslumenn.client.ts

```diff
@@ -20,30 +26,57 @@ export class SyslumennClient {
     private clientConfig: SyslumennClientConfig,
   ) {}
 
-  private async getToken() {
+  private async login() {
     const config = {
       notandi: this.clientConfig.username,
       lykilord: this.clientConfig.password,
     }
 
     const response: { data: ILogin } = await this.httpService
-      .post(`${this.clientConfig.url}/dev/v1/Innskraning`, config)
+      .post(`${this.clientConfig.url}/v1/Innskraning`, config)
       .toPromise()
 
-    this.accessToken = response.data.audkenni
+    this.id = response.data.audkenni
+    this.accessToken = response.data.accessToken
```

Comment:
FYI: Since the login response returns a value called `accessToken` which I have to use to upload data then I changed the variable name of the variable formerly known as `accessToken` in this scope to `id`